.github/workflows/autoupdate.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     name: autoupdate
     runs-on: ubuntu-20.04
     steps:
-      - uses: docker://chinthakagodawita/autoupdate-action:v1
+      - uses: docker://chinthakagodawita/autoupdate-action:v1.7
```

Comment:
Could you bump the ubuntu version also to `22.04`? This is what's in the README: https://github.com/chinthakagodawita/autoupdate